script/vsts/platforms/windows.yml

```diff
@@ -1,5 +1,6 @@
 jobs:
-  - job: Windows
+  - job: Windows_build
+    displayName: Windows build
```

Comment:
Please make this `Windows` to keep it simple

Reply:
I was copying the `macOS build` upstream uses for macOS, where the CI is split up between a "build" and a "parallel test" job. Just like we introduced for Windows at this fork.

So maybe upstream will eventually take this? I do think they want parallel testing on Windows x64. (I think they might be more likely to take it if it is separated out from the "move lots of stuff to templates" changes. But I'm not them so I can't read their minds about that.)

Reply:
This job is called on other files too like Release job. After all, it is just a name, I don't think it is important. 😄 If you want to change it you change it everywhere.